### PR TITLE
explicit warning regarding upgradable versions

### PR DESCRIPTION
**Pull request by mmoll**
once this receives a positive review, I'll apply that change to all versions of the manual.

### CODE REVIEW

File: _includes/manuals/1.10/3.6_upgrade.md [hunk 1]
```diff
@@ -3,9 +3,10 @@
 
 #### Preparation
 
-When upgrading across more than one version it is recommended to upgrade to each
-intermediate version and follow all upgrade instructions for the previous versions
-before updating to {{page.version}}.
+**Before updating to {{page.version}}, make sure you successfully upgraded to {{page.previous_version}} before.**
```
Comment (domcleal) [inline _includes/manuals/1.10/3.6_upgrade.md]: Slight grammar note - "make sure you _have_ successfully", s/before/first/

**Review by domcleal — CHANGES_REQUESTED** [review]
Looks fine, I wouldn't bother updating older manuals.

**Comment by mmoll** [comment]
done

File: manuals/1.3/quickstart_guide.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 layout: manual
 title: Foreman Quickstart Guide
 version: "1.3"
+previous_version: "1.2"
```
Comment (domcleal) [inline manuals/1.3/quickstart_guide.md]: This should be in index.md, not quickstart_guide.md.

**Review by domcleal — CHANGES_REQUESTED** [review]

File: manuals/1.3/quickstart_guide.md (hunk 2 above)
Comment (mmoll) [reply manuals/1.3/quickstart_guide.md]: whoops, fixed

**Comment by domcleal** [comment]
Thanks @mmoll.

**Pull request merged**